### DISCUSSION TITLE
Uploading files

### DISCUSSION THREAD

**Discussion by XeR** [TFNS/CTFNote#96] [Ideas]
As mentioned in issue #65, it might be interesting to add a way to upload file.

This could be used to:
- upload logos for non-CTFtime CTF
- upload screenshots for Hedgedoc
- upload files to a task

How do we want to upload files ?
Do you have additional feature ideas ?

**Comment by XeR** [comment]
We can upload logos since 599f2959244194e6a76cd21781499dc92bc7a0da
Issue #65 mentioned [MinIO](https://docs.minio.io/), but we did not look into it at all

**Comment by JJ-8** [comment]
When uploading files is being implemented, it is also very interesting to re-upload the files send through Discord to the CTFNote instance since Discord download links will expire at some point. So therefore during archiving the content can be stored permanently.
